forage-android/src/main/java/com/joinforage/forage/android/core/ui/element/ForageElement.kt

```diff
@@ -24,22 +25,11 @@ import com.joinforage.forage.android.core.ui.element.state.ElementState
 data class ForageConfig(
     val merchantId: String,
     val sessionToken: String
-)
+) {
+    internal val envConfig = EnvConfig.fromForageConfig(this)
+}
 
-/**
- * The interface that defines methods for configuring and interacting with a [ForageElement].
- * A ForageElement is a secure, client-side entity that accepts and submits customer input for a
- * transaction.
- * Both [ForagePANEditText] and [ForagePINEditText] adhere to the [ForageElement] interface.
- *
- * @property typeface The [Typeface](https://developer.android.com/reference/android/graphics/Typeface)
- * that is used to render text within the ForageElement.
- * @see * [Online-only Android Quickstart](https://docs.joinforage.app/docs/forage-android-quickstart)
- * * [POS Terminal Android Quickstart](https://docs.joinforage.app/docs/forage-terminal-android)
- * * [Guide to styling Forage Android Elements](https://docs.joinforage.app/docs/forage-android-styling-guide)
- */
-interface ForageElement<T : ElementState> {
-    var typeface: Typeface?
+interface DynamicEnvElement {
```

Comment:
Can this be internal?

Reply:
Oooooo and, on that note, we can actually make all of these internal:
* `DynamicElement`
* `EditTextElement`
* `ForageElement<T : ElementState>`
which would be a nice win for reducing our public contract. Great callout, Danilo!

Reply:
OK - all three have been marked `internal` with the last commit!